### DISCUSSION TITLE
有教程吗

### DISCUSSION THREAD

**Discussion by adaozi** [nkxingxh/MiraiEz#11] [Q&A] [answered]
纯纯萌新，一点都不会，看了csdn上的教程搭建过后，不回消息这是为什么，希望能给个教程

**Comment by nkxingxh** [comment]
你配置好mirai-api-http与config.php就可以了。然后去看示例插件

**Reply by adaozi** [reply]
配置好了，但是框架收到了消息，但是机器人不回复是为啥
![Snipaste_2023-01-07_12-25-42](https://user-images.githubusercontent.com/99073253/211179136-71758071-de75-4b67-b7cb-976e9f97510f.png)
![Snipaste_2023-01-07_12-26-15](https://user-images.githubusercontent.com/99073253/211179139-cd4f6103-6f57-475f-86ca-ede2bbfe8d77.png)

**Comment by nkxingxh** [comment, marked as answer]
你这大概率是没配置好。参考排查教程
https://blog.nkxingxh.top/archives/234/

**Comment by MR-XieXuan** [comment]
@adaozi 可能是由于 Mirai-api-http 的适配器存在错误。我在CSDN上没有写到这个问题。你可以对照我在哔哩哔哩发布的视频进行配置。或者获取最新的 MiraiEz 版本。根据 ReadMe 中的指引。强制使用 HTTP 进行回复。
